lib/src/facts/windows/uptime_resolver.cc

```diff
@@ -1,4 +1,5 @@
 #include <internal/facts/windows/uptime_resolver.hpp>
+#include <leatherman/windows/windows.hpp>
 #include <leatherman/windows/wmi.hpp>
```

Comment:
There's some larger cleanup that could happen here: almost all the includes could be removed, wmi argument to uptime_resolver can be dropped.